sql/src/main/java/io/druid/sql/calcite/table/RowSignature.java

```diff
@@ -76,6 +76,21 @@ public static Builder builder()
     return new Builder();
   }
 
+  public static ValueType getValueTypeForSqlTypeName(SqlTypeName sqlTypeName)
```

Comment:
Please move this to "Calcites" where most of the random utility functions should go.